Content.Client/Popups/PopupSystem.cs

```diff
@@ -123,6 +123,12 @@ public override void PopupCoordinates(string? message, EntityCoordinates coordin
                 PopupMessage(message, type, coordinates, null, true);
         }
 
+        public override void PopupPredictedCoordinates(string? message, EntityCoordinates coordinates, EntityUid? recipient, PopupType type = PopupType.Small)
+        {
+            if (recipient != null && _timing.IsFirstTimePredicted)
```

Comment:
Oh wait, I missed something.
Shouldn't this check for the correct recipient instead of any recipient?
```suggestion
            if (_playerManager.LocalSession == recipient & _timing.IsFirstTimePredicted)
```

Reply:
Thanks!